webapp/channels/src/components/channel_layout/center_channel/center_channel.tsx

```diff
@@ -115,7 +116,7 @@ export default class CenterChannel extends React.PureComponent<Props, State> {
                             component={Drafts}
                         />
                         <Route
-                            path={`/:team(${TEAM_NAME_PATH_PATTERN})/scheduled_posts`}
```

Comment:
did you add this path to the `doesRouteBelongToTeamControllerRoutes`

Reply:
Oh, I didn't knew that. Added now. I've used the variable instead of inserting the URL suffix in there, so please take a look.

Reply:
LGTM